### DISCUSSION TITLE
How to set `processors` in classic mode configuration format?

### DISCUSSION THREAD

**Discussion by sh-cho** [fluent/fluent-bit#8338] [Q&A] [answered]
https://github.com/fluent/fluent-bit/blob/97e2d20c23da61b87d48ba5aad6804c2786ba32f/tests/internal/data/config_format/yaml/processors.yaml#L5-L10

I just wonder if processor config is possible only in yaml config.

**Comment by arapaho** [comment, marked as answer]
Unfortunately, this configuration parameter is only available while using yaml config.  
From the [processors documentation](https://docs.fluentbit.io/manual/administration/configuring-fluent-bit/yaml/configuration-file#config_processors): "This functionality is only exposed in YAML configuration and not in classic configuration mode due to the restriction of nested levels of configuration."
